.mergify.yml

```diff
@@ -24,7 +24,7 @@ pull_request_rules:
         rebase_fallback: none
   - name: L10N - Auto Merge
     conditions:
-      - author=mozilla-l10n-automation-bot
+      - author~=(mozilla-l10n-automation-bot|github-actions\[bot\])
```

Comment:
We still want `mozilla-l10n-automation-bot` around because this account is still used on the `main` branch. `github-actions[bot]` is active on the release ones.

[1] https://github.com/mozilla-mobile/android-components/pull/12247